src/pages/user-details/user-details.scss

```diff
@@ -1,3 +1,6 @@
 page-user-details {
 
 }
+ion-icon{
```

Comment:
Can you, please, move this part inside page-user-details descriptor? Otherwise, it may produce a bug by applying this style to other ion-icon descriptors that are not on this page.
You can refer this link: https://www.joshmorony.com/a-guide-to-styling-an-ionic-2-application/